internal/rbd/nodeserver.go

```diff
@@ -69,12 +69,20 @@ var (
 	deepFlattenSupport = []util.KernelVersion{
 		{
 			Version:      5,
-			PatchLevel:   2,
+			PatchLevel:   1,
```

Comment:
Yeah, 5.2 is incorrect, not sure where it came from.

You may want to update `internal/util/util_test.go` too -- it's just a test (which looks redundant to me), but it also references 5.2.

Reply:
Yeah, it's just a test as 5.2 is greater than 5.1 it will pass. will correct it later as it should not block this one.